apis/swagger.yml

```diff
@@ -3597,6 +3597,49 @@ definitions:
           items:
             type: "string"
 
+  EventsMessage:
+    description: "represents the information an event contains"
+    type: "object"
+    properties:
+      status:
+        type: "string"
+      id:
+        type: "string"
+      from:
+        type: "string"
+      type:
+        $ref: "#/definitions/EventType"
+      action:
+        type: "string"
+      actor:
+        $ref: "#/definitions/EventsActor"
+      time:
+        type: "integer"
+      timeNano:
+        type: "integer"
+
+  EventsActor:
+    description: |
+      EventsActor describes something that generates events,
+      like a container, or a network, or a volume.
+      It has a defined name and a set or attributes.
+      The container attributes are its labels, other actors
+      can generate these attributes from other properties.
+    type: "object"
+    properties:
+      ID:
+        type: "string"
+      Attributes:
+        type: "object"
+        additionalProperties:
+          type: "string"
+
+  EventType:
+    description: the type of event. For example, "container" or "image"
+    type: "string"
+    enum: ["container", "daemon", "image", "network", "plugin", "volume"]
```

Comment:
We have already implemented all these object's events? Just double check.
Otherwise, I encourage that we add more information about which ones are TODO.